pages/api/getData.js

```diff
@@ -1,6 +1,3 @@
-import fs from 'fs'
-import path from 'path'
```

Comment:
Maybe comment this lines instead or removing the function that loads data from local files ??

Reply:
Added them as comments :)